string type
geometry_msgs/Polygon geometry
# z coordinate will be stripped
